nncf/onnx/graph/model_transformer.py

```diff
@@ -31,32 +29,72 @@
 from nncf.common.graph.model_transformer import ModelTransformer
 
 
-# pylint: disable=no-member
 class ONNXModelTransformer(ModelTransformer):
+    """
+    Applies transformations upon ONNX model.
+    ModelTransformer should be created once for a particular model,
+    and be used to apply transformations to the provided model.
+    """
+
     QUANTIZER_NAME_PREFIX = 'QuantizeLinear_'
     DEQUANTIZER_NAME_PREFIX = 'DequantizeLinear_'
     SCALE_TENSOR_NAME_PREFIX = 'scale_'
     ZERO_POINT_NAME_PREFIX = 'zero_point_'
 
     def __init__(self, model: onnx.ModelProto):
         super().__init__(model)
-        self._model = deepcopy(model)
-        self._nncf_graph = NNCFGraphFactory.create(self._model)
+        nncf_graph = NNCFGraphFactory.create(self._model)
```

Comment:
Because ModelTransformer applies to one particular model `self._model`, which is unchanged, so we can get the model information in initialization and utilize at any transformation.

Reply:
Why we can't do the same for the `OpenVINO` backend then? It's not clear to me, why the behavior of `ModelTransformer` differs from backend to backend. The main problem with such a difference is that we can't fully predict how to use the `ModelTransformer` in the shared part of the code.